Makefile

```diff
@@ -329,6 +322,12 @@ else
 endif
 
 purge: purge-doc purge-dev user-uninstall
+ifneq ($(PREFIX),/usr)
+	PREFIX=/usr
```

Comment:
what are you trying to do in here? its dead code , that is unused and it does nothing because it will be freed at the end of the line